lib/internal/bootstrap_node.js

```diff
@@ -7,9 +7,13 @@
 
 'use strict';
 
-(function(process) {
+(function(global, process) {
+
+  function startup(global, process) {
+    // Expose the global object as a property on itself
+    // (Allows you to set stuff on `global` from anywhere in JavaScript.)
```

Comment:
I didn't write this comment, I only moved it around.  I can remove it if you want.